src/lib/components/input/QInput.svelte

```diff
@@ -21,6 +21,7 @@
     label = undefined,
     outlined = false,
     rounded = false,
+    type= undefined,
```

Comment:
```suggestion
    type = undefined,
```
Could you add a space please?